test/server/config/network/config_test.cc

```diff
@@ -532,6 +539,36 @@ TEST(TcpProxyConfigTest, TcpProxyConfigTest) {
   cb(connection);
 }
 
+TEST(NetworkFilterConfigTest, ExtAuthzCorrectProto) {
+  std::string json = R"EOF(
+    {
+      "grpc_service": {
+          "google_grpc": {
+             "target_uri": "ext_authz_server",
+             "stat_prefix": "google"
+           }
+      },
+      "failure_mode_allow": false,
+      "stat_prefix": "name"
+    }
+    )EOF";
+
+  envoy::config::filter::network::ext_authz::v2::ExtAuthz proto_config{};
+  MessageUtil::loadFromJson(json, proto_config);
```

Comment:
FYI, we now generally prefer YAML examples and using  `MessageUtil::loadFromYaml`, but we can fix this later or leave it as is, I'd like to unblock your other PR review.